frontend/src/store/helpers.js

```diff
@@ -1,6 +1,17 @@
-import moment from 'moment'
 import { months as m, weekDays as wd } from './constants';
 
+Date.prototype.getWeek = function () {
+  var target  = new Date(this.valueOf());
+  var dayNr   = (this.getDay() + 6) % 7;
```

Comment:
Change `var` to `const`. Don't align equal signs.

Reply:
@fsandelin